### ISSUE TITLE
有关intermediate supervision的问题

### ISSUE THREAD

**Issue by BananaLv26** [LiMeng95/MultiPoseNet.pytorch#27]
为什么代码里面计算intermediate supervision 的时候计算heatmap通道数是19？和GT对应的话不是应该18吗
self.convfin_k2 = nn.Conv2d(256, 19, kernel_size=1, stride=1, padding=0)
self.convfin_k3 = nn.Conv2d(256, 19, kernel_size=1, stride=1, padding=0)
self.convfin_k4 = nn.Conv2d(256, 19, kernel_size=1, stride=1, padding=0)
self.convfin_k5 = nn.Conv2d(256, 19, kernel_size=1, stride=1, padding=0)

**Comment by accountcwd** [comment]
多出来的通道应该是做分割的mask，详细看[这里的loss定义](https://github.com/LiMeng95/MultiPoseNet.pytorch/blob/7b548fc20ae5b0141a2113669e8606103fb7e6d4/network/posenet.py#L381)，不过貌似大佬屏蔽掉了，应该没做好吧。

**Comment by BananaLv26** [comment]
> 多出来的通道应该是做分割的mask，详细看[这里的loss定义](https://github.com/LiMeng95/MultiPoseNet.pytorch/blob/7b548fc20ae5b0141a2113669e8606103fb7e6d4/network/posenet.py#L381)，不过貌似大佬屏蔽掉了，应该没做好吧。

好的，感谢！
